### ISSUE TITLE
Support for 2021.1 EAP

### ISSUE THREAD

**Issue by Sxtanna** [Oberdiah/ClassPlane#1]
Fails to install on IntelliJ IDEA 2021.1 EAP (Ultimate Edition) Build #IU-211.5538.20

![image](https://user-images.githubusercontent.com/20979389/107500607-5c0a1b80-6b64-11eb-8154-7e121c3c1a16.png)

**Comment by Oberdiah** [comment]
I imagine the plugin will run fine on the new EAP versions, but it's not listed as such because I tend to develop on the stable builds. It should be possible to build it on a newer version to try it out, though, if you wanted.

**Comment by Sxtanna** [comment]
Yes, before I made this issue I manually updated the plugin.xml to have the EAP build number set for the `until` value. It works perfectly.

Here is what I set it to for reference:
```xml
<idea-version since-build="203.5981" until-build="211.*"/>
```

Just thought I should still make this issue.
